src/app/core/nav-menu/components/app-nav/app-nav.component.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <cs-menu-header (buttonClicked)="close.emit(true)"></cs-menu-header>
```

Comment:
Is this the agreement, that you have to emit event right from the template? I don't really like when internal logic of events handling is included in the template. Imagine that one day you'll decide to change event emitter to Subject, in this case, you have also change the template.

We prefer to put such logic inside of public methods of the component, like `public open() { this.close.emit(true) }`. Also, it will allow you to check that `this.close` actually initiated and exists.